ouroboros-consensus/test-consensus/Test/Dynamic/General.hs

```diff
@@ -220,3 +262,53 @@ prop_general k TestConfig{numSlots, nodeJoinPlan, nodeTopology} schedule
       where
         NumSlots t = numSlots
         NodeJoinPlan m = nodeJoinPlan
+
+    -- check for Chain Growth violations if there are no Common Prefix
+    -- violations
+    --
+    -- We consider all possible non-empty intervals, so the interval span
+    -- @s@ varies but is always at least 1. We compute a different /speed
+    -- coefficient/ @τ@ for each interval under the assumption that there are
+    -- no message delays (ie @Δ = 0@). This is essentially a count of the
+    -- active slots for that interval in the refined @schedule'@.
+    prop_all_growth =
+        (.||.) (not isConsensusExcepected) $
+        conjoin $ zipWith f slotDepths (List.tails slotDepths)
+      where
+        f ::
+             (SlotNo, BlockNo, BlockNo)
+          -> [(SlotNo, BlockNo, BlockNo)]
+          -> Property
+        f (s1, _, max1) =
+            conjoin .
+            map (\(s2, min2, _) -> prop_growth (s1, max1) (s2, min2))
+
+        prop_growth :: (SlotNo, BlockNo) -> (SlotNo, BlockNo) -> Property
+        prop_growth (s1, b1) (s2, b2) =
+            counterexample (condense (s1, s2, b1, b2, numActiveSlots)) $
+            (.&&.)
```

Comment:
Same comment about prefix vs infix here.

Reply:
Done; named each conjunct in the `where`